cypress/features/regression/designSystem/menu.feature

```diff
@@ -42,4 +42,44 @@ Feature: Design Systems Menu component
     When I hover over third expandable Menu component
     Then "fourth" submenu has alternate colour theme
       And "fifth" submenu has alternate colour theme
-      And "sixth" submenu has alternate colour theme
\ No newline at end of file
+      And "sixth" submenu has alternate colour theme
+
+  @negative
+  Scenario: Check the default menu clickToOpen element does not open on hover
+    Given I open "Design System Menu" component page "submenu options" in no iframe
+    When I hover over default menu fifth expandable Menu component
+    Then Menu fifth expandable element submenu is not visible
+
+  @positive
+  Scenario: Check the default menu clickToOpen element opens on mouse click
+    Given I open "Design System Menu" component page "submenu options" in no iframe
+    When I click default menu fifth expandable Menu component
+    Then Menu fifth expandable element has inner elements
+
+  @positive
+  Scenario: Check the default menu clickToOpen element opens when Enter is pressed
```

Comment:
we can combine all those TC below into one and just use `Examples:` table and provide different `keys` to press